kuksa-client/kuksa_client/cli_backend/grpc.py

```diff
@@ -36,9 +37,15 @@
 from kuksa.val.v1 import types_pb2
 
 
+logger = logging.getLogger(__name__)
+
+
 def callback_wrapper(callback: Callable[[str], None]) -> Callable[[Iterable[EntryUpdate]], None]:
     def wrapper(updates: Iterable[EntryUpdate]) -> None:
-        callback(json.dumps([update.to_dict() for update in updates], cls=DatabrokerEncoder))
+        try:
+            callback(json.dumps([update.to_dict() for update in updates], cls=DatabrokerEncoder))
+        except Exception as e:
+            logger.error("Callback could not be executed", e)
```

Comment:
This is an improvement related to #27 - give an error if we by some reason cannot dump the response as json.